static/app/views/performance/transactionSummary/transactionOverview/index.tsx

```diff
@@ -131,6 +132,11 @@ function OverviewContentWrapper(props: ChildProps) {
     referrer: 'api.performance.transaction-summary',
   });
 
+  useEffect(() => {
+    const isMetricsData = getIsMetricsDataFromResults(queryData.data);
+    mepContext.setIsMetricsData(isMetricsData);
+  }, [mepContext, queryData.data]);
```

Comment:
This is what's setting the metrics fallback, we _could_ instead track every query for metrics data fallback, but this totals query hits all of the metrics tables so it's all we should need, in theory, at least.